### DISCUSSION TITLE
How to easily retrieve point coordinates?

### DISCUSSION THREAD

**Discussion by StoneIslandFlexBoy** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#262] [Q&A] [answered]
Hey, first of all thanks for this nice card.

I tried to create some rooms and read the tipps you gave about retrieving coordinates. It all works so far but i dont really understand what you mean by "Your outline will be in path section of service data". Is service data a file? or where exacactly can i find the outline coordiantes. I do get the popup aswell but it cannot display all coordinates and ends with "..." at some point.

Thanks already in advance.

**Comment by StoneIslandFlexBoy** [comment, marked as answer]
nevermind, i used the js console with "ctrl + shift + j" on windows, its written there aswell. Thanks

**Reply by murphy2712** [reply]
You can long press on the play zone button, or double tap on it and it will show in a popup or the browser console.
